Recon/src/Recon/Event/EventConstants.cxx

```diff
@@ -2,11 +2,6 @@
 
 namespace ldmx {
 
-/*
- * Default name of event tree.
- */
-const std::string EventConstants::EVENT_TREE_NAME = "LDMX_Events";
```

Comment:
I think we should delete this whole file...

Reply:
We should, but that should be done in a different PR.